src/cloudservice/azext_cloudservice/aaz/latest/cloud_service/_reimage.py

```diff
@@ -0,0 +1,186 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "cloud-service reimage",
```

Comment:
May I ask if we have added test for this new added command?

Reply:
It is available in cmd, but there are no resources to test it.